scripts/test.mjs

```diff
@@ -37,4 +37,9 @@ for (const projectName of projects) {
     console.log(`Running unit tests in ${projectName}`)
     await $`pnpm test:unit`
   }
+
+  if ('test:unit' in packageJSON.scripts || 'test:e2e' in packageJSON.scripts) {
+    console.log(`Running the test script in ${projectName}`)
+    await $`pnpm test`
+  }
```

Comment:
I think this can be removed as the script already tests the unit and e2e tests, and it already takes a _really long_ time to execute locally.